### PR TITLE
Feature/projects search

### PR DESCRIPTION
**Pull request by NikitaChar**
This PR includes the search page for institutions, teams, user and projects. 
Search bar and filter are currently primitive but will be integrated with f2-components asap.
Screenshot from 2022-04-29 13-36-20

### CODE REVIEW

File: Frontend/src/components/Search/SearchContent/index.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  return (
+    <React.Fragment>
+      <ResultStatus
+        count={searchedData.length}
+        searchedEntity={type}
+        searchedKeyword={searchedKeyword}
+      />
+      <Switch>
+        <Route path={setTypeInUrl(path, "teams")}>
```
Comment (viganll) [inline Frontend/src/components/Search/SearchContent/index.jsx:34]: Why do not you build up the string directly? but you use a function for it. You can leave it as it is but just wanted to raise it.

**Review by viganll — COMMENTED** [review]
I probably take a look again after you merge your changes with frontend 2 changes.

File: Frontend/src/components/Search/Filter/TeamsFilter/index.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+// Teams Filter needed only in the team category
+const TeamsFilter = ({ visibility, onChange }) => {
+  const visibilityItems = [
+    { label: "All", value: "" },
+    { label: "Public", value: "public" },
+    { label: "Private", value: "private" },
+    { label: "By institution", value: "byInstitution" },
+  ];
+
+  useEffect(
```
[the remark this answers is not in the document]
Comment (NikitaChar) [reply Frontend/src/components/Search/Filter/TeamsFilter/index.jsx:13]: Note, the function returns a function that will be executed on component dismount. Goal is to clean up the parameters from the url that are not relevant for other filters to keep it consistent. (e.g. "visibility" is Team filter specific not needed in other filters)

File: Frontend/src/components/Search/SearchContent/index.jsx (hunk 1 above)
Comment (NikitaChar) [reply Frontend/src/components/Search/SearchContent/index.jsx:34]: So my goal was avoid hardcoding <Route path="..."/> so if the route managed in DashbordContent is changed, the component is not/less affected. Maybe there is a cleaner/in-build solution for that, but I couldn't find it.

**Comment by NikitaChar** [comment]
Current state with new F2 components
Screenshot from 2022-05-01 15-17-42

**Review by viganll — APPROVED** [review]

**Pull request merged**